ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml

```diff
@@ -87,6 +87,46 @@ tests:
       SCENARIO: ci-10u-10t
     workflow: redhat-appstudio-load-test
   timeout: 8h0m0s
+- always_run: false
+  as: load-test-ci-10u-10t-go
+  steps:
+    cluster_profile: aws
+    env:
+      SCENARIO: ci-10u-10t-go
+    workflow: redhat-appstudio-load-test
+  timeout: 8h0m0s
+- always_run: false
+  as: load-test-ci-10u-10t-java
+  steps:
+    cluster_profile: aws
+    env:
+      SCENARIO: ci-10u-10t-java
+    workflow: redhat-appstudio-load-test
+  timeout: 8h0m0s
+- always_run: false
+  as: load-test-ci-10u-10t-nodejs
+  steps:
+    cluster_profile: aws
+    env:
+      SCENARIO: ci-10u-10t-nodejs
+    workflow: redhat-appstudio-load-test
+  timeout: 8h0m0s
+- always_run: false
+  as: load-test-ci-10u-10t-python
+  steps:
+    cluster_profile: aws
+    env:
+      SCENARIO: ci-10u-10t-python
+    workflow: redhat-appstudio-load-test
+  timeout: 8h0m0s
+- always_run: false
+  as: load-test-ci-10u-10t-quarkus
```

Comment:
@naftalysh This `load-test-ci-10u-10t-quarkus` job is effective duplicate of the `load-test-ci-10u-10t` job